javascript-functions/main.js

```diff
@@ -0,0 +1,33 @@
+const convertMinutesToSeconds = minutes => {
+  const seconds = minutes * 60;
+  console.log(`${minutes} minutes is ${seconds} seconds`);
```

Comment:
so instead of console logs, you want to make sure that each of your functions returns a value.
so to complete each of these, you want to return the proper value and store that in a variable, then log out the variable to see the contents